jetson/nav/search/searchStateMachine.cpp

```diff
@@ -354,6 +417,26 @@ void SearchStateMachine::insertIntermediatePoints()
     }
 } // insertIntermediatePoints()
 
+bool SearchStateMachine::checkTurnStatus() {
+    double updateBearing = ( mRover->roverStatus().leftTarget().distance == -1 
```

Comment:
here again is there a reason not to just use the cached target's bearing?

Reply:
Thought about it and honestly you are right, caches will be up to date regardless, so let's just replace all instances of leftTarget/rightTarget with their cached ones; I just included them to be sure of any inconsistencies (worst case if this fails, we add them back even if it's ugly)